README.md

```diff
@@ -48,6 +48,77 @@ To read the encrypted NFC data, call `readNfc()`, making sure to call this metho
 val passportInfo = nfcReader.readNfc(tag, mrzInfo)
 ```
 
+## Processing live camera input
+
+Hooking up the `MrzProcessor` to process each frame of camera input will depend on the camera library used
+
+### CameraX
+To process each camera frame, make use of `ImageAnalysis`
+```kotlin
+val imageAnalysis = ImageAnalysis.Builder()
+    .setOutputImageFormat(OUTPUT_IMAGE_FORMAT_YUV_420_888)
+    .setTargetResolution(Size(1280, 720))
+    .setBackpressureStrategy(STRATEGY_KEEP_ONLY_LATEST)
+    .build()
+```
+After creating the `ImageAnalysis` object, set an `Analyzer` on it
+```kotlin
+imageAnalysis.setAnalyzer(cameraExecutor) { imageProxy: ImageProxy ->
+    mrzProcessor.processImage(imageProxy.toCameraImage(), object : MrzProcessorResultListener {
+        override fun onSuccess(mrzInfo: MrzInfo) {
```

Comment:
In our sample yes because that is our use case, but it will be up to the calling app when they want to remove the analyzer. For example, they may only want to draw on top of the preview instead of immediately navigating away from the camera. `ImageAnalysis.clearAnalyzer` should only be called when we want to _stop_ analyzing input frames. If you prefer we could replace the `startActivity()` call with a comment like `// do something with MrzInfo`, what do you think? 🤔